web/regression/javascript/erd/ui_components/ERDTool.spec.js

```diff
@@ -1,505 +1,472 @@
-    let newData = {
-      local_table_uid: 'id1',
-      local_column_attnum: 1,
-      referenced_table_uid: 'id2',
-      referenced_column_attnum: 2,
-    };
-    saveCallback(newData);
-    expect(bodyInstance.diagram.addLink).toHaveBeenCalledWith(newData, 'onetomany');
-  });
-
-  it('onManyToManyClick', ()=>{
-    let node = new FakeNode({}, 'id1');
-    let node1 = new FakeNode({'name': 'table1', schema: 'erd1', columns: [{name: 'col1', type: 'type1', attnum: 1}]}, 'id1');
-    let node2 = new FakeNode({'name': 'table2', schema: 'erd2', columns: [{name: 'col2', type: 'type2', attnum: 2}]}, 'id2');
-    let nodesDict = {
-      'id1': node1,
-      'id2': node2,
-      'newid1': newNode,
-    };
-    spyOn(bodyInstance.diagram, 'getModel').and.returnValue({
-      'getNodesDict': ()=>nodesDict,
-    });
-    spyOn(bodyInstance.diagram, 'getSelectedNodes').and.returnValue([node]);
-
-    bodyInstance.onManyToManyClick();
-
-    /* onSave */
-    spyOn(bodyInstance.diagram, 'addLink');
-    let saveCallback = mtmDialog.calls.mostRecent().args[2];
-    let newData = {
-      left_table_uid: 'id1',
-      left_table_column_attnum: 1,
-      right_table_uid: 'id2',
-      right_table_column_attnum: 2,
-    };
-
-    bodyInstance.diagram.addNode.calls.reset();
-    bodyInstance.diagram.addLink.calls.reset();
-    saveCallback(newData);
-    let tableData = bodyInstance.diagram.addNode.calls.argsFor(0)[0];
-    expect(tableData).toEqual(jasmine.objectContaining({
-      name: 'table1_table2',
-      schema: 'erd1',
-    }));
-    expect(tableData.columns[0]).toEqual(jasmine.objectContaining({
-      type: 'type1',
-      name: 'table1_col1',
-      attnum: 0,
-    }));
-    expect(tableData.columns[1]).toEqual(jasmine.objectContaining({
-      type: 'type2',
-      name: 'table2_col2',
-      attnum: 1,
-    }));
-
-    let linkData = {
-      local_table_uid: 'newid1',
-      local_column_attnum: 0,
-      referenced_table_uid: 'id1',
-      referenced_column_attnum : 1,
-    };
-    expect(bodyInstance.diagram.addLink.calls.argsFor(0)).toEqual([linkData, 'onetomany']);
-    linkData = {
-      local_table_uid: 'newid1',
-      local_column_attnum: 1,
-      referenced_table_uid: 'id2',
-      referenced_column_attnum : 2,
-    };
-    expect(bodyInstance.diagram.addLink.calls.argsFor(1)).toEqual([linkData, 'onetomany']);
-  });
-
-  it('onNoteClick', ()=>{
-    let noteNode = {key: 'value', getNote: ()=>'a note'};
-    spyOn(bodyInstance.diagram, 'getSelectedNodes').and.returnValue([noteNode]);
-    spyOn(bodyInstance.diagram.getEngine(), 'getNodeElement').and.returnValue(null);
-    spyOn(bodyInstance.diagram.getEngine(), 'getNodeElement').and.returnValue(null);
-    spyOn(bodyInstance, 'setState');
-    bodyInstance.onNoteClick();
-    expect(bodyInstance.setState).toHaveBeenCalledWith({
-      note_node: noteNode,
-      note_open: true,
-    });
-  });
+  it('dummy', ()=>{});
 });
+
+// import React from 'react';
```

Comment:
Remove comments.

Reply:
This particular test file is intentionally commented and I have mentioned the reason in the same file.